azure-pipelines.yml

```diff
@@ -123,7 +123,7 @@ jobs:
           # There's no easy way to get the test results in machine-readable
           # form to properly exclude known-failing tests. Going with an
           # approximate solution for now.
-          if [ $isa == rv32i ] && grep -q 'FAIL: 5/55' run.log; then
+          if [ $isa == rv32i ] && grep -q 'FAIL: 4/55' run.log; then
```

Comment:
This already change in the master.
We can now add `rv32Zifencei` as a make target to the `isa` loop.